r/R/csv.R

```diff
@@ -22,8 +22,8 @@
 #' `readr::read_delim()`, and `col_select` was inspired by `vroom::vroom()`.
 #'
 #' `read_csv_arrow()` and `read_tsv_arrow()` are wrappers around
-#' `read_delim_arrow()` that specify a delimiter. `read_csv2_arrow()` uses `;`⁠ for
-#' the delimiter and `,` for the decimal point.
+#' `read_delim_arrow()` that specify a delimeter. `read_csv2_arrow()` uses `;` for
```

Comment:
I was corrected on this by someone else when working on this recently, but I believe that "delimiter" is the correct spelling here rather than "delimeter"?

Reply:
Sorry, didn't realise this has merged, I'll open a PR!